### PR TITLE
fastboot compatibility

### PR DESCRIPTION
**Pull request by bryn19**
go fast

jira: https://jira.wnyc.org/browse/MT-147

### CODE REVIEW

File: addon/adapters/comment.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import config from 'ember-get-config';
 import DS from 'ember-data';
-import wrapAjax from 'nypr-django-for-ember/utils/wrap-ajax'
 // TODO: auth headers for native fetch
-// import fetch from 'fetch';
+import fetch from 'fetch';
```
Comment (kedmundson) [inline addon/adapters/comment.js]: stop trying to make fetch happen

File: addon/components/story-comments.js [hunk 2]
```diff
@@ -41,6 +42,10 @@ export default Component.extend({
 
   isShowingComments: computed({
     get() {
+      if (typeof document === 'undefined') {
+        // don't run in fastboot
```
Comment (kedmundson) [inline addon/components/story-comments.js]: So do we lose comments in fastboot? What impact does this have?

Comment (kedmundson) [reply addon/components/story-comments.js]: or are we now pre-fetching them in the comment adapter above

Comment (bryn19) [reply addon/components/story-comments.js]: we will still have them, this variable just wont be set upon the initial pre-render.

**Pull request merged**